app/components/Question/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+/* component styles */
+import { styles } from './styles.scss';
+
+const Question = ({question}) => {
```

Comment:
I find this new'ish React way to create stateless components by using plain ES6 arrow functions to be quite obscure. I feel like they should be annotated with a comment `/* Create stateless react component */` so it's easier to realize instantly you are looking at a React render method.